### ISSUE TITLE
Enable restart of Hercules

### ISSUE THREAD

**Issue by genevievestarke** [NREL/hercules#75]
Add capability to use Hercules (and AMR) output files to restart the simulation and decide on an output structure to support this

**Comment by paulf81** [comment]
@genevievestarke propose to close this one for v2

**Comment by genevievestarke** [comment]
I agree, now that Hercules runs faster I don't think this one is as pressing
